src/demo.js

```diff
@@ -1,5 +1,6 @@
 import { createApp } from 'vue'
 import Demo from './Demo.vue'
+import A11yDialog from '../dist/vue-a11y-dialog.esm.js'
```

Comment:
This may be a named export 🤔  e.g. `import { A11yDialog }..`

Reply:
Btw, I really like pulling from `../dist/vue-a11y-dialog.esm.js` which hammers on our build system to better verify the distributable is in fact working -- great idea @morkro 👍

Reply:
Ok, sorry for back and forth just contemplating and realized, we can leave this here for use case that someone wants to start this outside of Cypress, but I believe it should use named import:

```js
import { createApp } from 'vue'
import Demo from './Demo.vue'
import { A11yDialog } from '../dist/vue-a11y-dialog.esm.js'
import './demo.css'

createApp(Demo).use(A11yDialog).mount('#app')
```

If you look at the generated esm build:

```js
...
var index = {
  install: (app) => {
    app.component("a11y-dialog", A11yDialog);
  }
};
export { A11yDialog, index as default };
```
So it looks like it exports both a named export and default. I've tested with my named import comments above and it works. But…

It seems like you should be able to use the `default` way too and it looks like it magically does the `install` thing and I guess that happens upon import before your application has mounted. But I can't get it to work after several various attempts. Only the named import works for me ¯\_(ツ)_/¯

Reply:
Actually, I suspect you have to call install yourself like this:

```js
import A11yDialog from '../dist/vue-a11y-dialog.esm.js'
import './demo.css'
const app = createApp(Demo)
A11yDialog.install(app)
app.mount('#app')
```

Reply:
In case it's easier to cherry-pick and/or reference a visual diff I've got this in the form of a commit that's pushed up to my repo's 1.1.0:

https://github.com/AgnosticUI/vue-a11y-dialog/commit/93daea064e6e71a9951e78d43de9cc8149c9fbfd

Reply:
Thanks for looking into this. Simply importing it directly in the component again, fixed the tests :)

Reply:
Soooooo close :)